pages/solid-events.md

```diff
@@ -14,7 +14,7 @@ Are you interested in speaking about what you have been building on Solid? Reach
 
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |---|-------|--------------|--------------|
-2020-12-03|[Solid World December](https://www.eventbrite.com/e/solid-world-tickets-128665029567)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/487088029)|
+2020-12-03|[Solid World December](https://www.eventbrite.com/e/solid-world-tickets-128665029567)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/487088029), The Community Server:(https://github.com/solid/community-server/), Ruben's live list: (https://rubenverborgh.github.io/solid-list-viewer/) with (https://css.verborgh.org/list)|
```

Comment:
```suggestion
2020-12-03|[Solid World December](https://www.eventbrite.com/e/solid-world-tickets-128665029567)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/487088029), The [Community Server](https://github.com/solid/community-server/), [Ruben's live list](https://rubenverborgh.github.io/solid-list-viewer/) with [his CSS Pod](https://css.verborgh.org/list)|
```
Without the square brackets, `(https://some.url)` renders as a link between parentheses in Markdown, while `[some text](https://some.url)` displays `some text` as a link. I assume the latter is what was intended here :)

Reply:
I think so too, because this is what [it currently looks like](https://deploy-preview-441--solidproject-dot-org.netlify.app/events):

![image](https://user-images.githubusercontent.com/4251/101478263-b6aead80-3950-11eb-8760-b96e47d4d114.png)
 :)